x-pack/plugins/cases/server/client/cases/bulk_update.ts

```diff
@@ -288,15 +288,18 @@ function partitionPatchRequest(
       conflictedCases.push(reqCase);
       // let's try to authorize the conflicted case even though we'll fail after afterwards just in case
       casesToAuthorize.set(foundCase.id, { id: foundCase.id, owner: foundCase.attributes.owner });
-    } else if (
+    } else {
```

Comment:
Could you please also update the tests in `x-pack/plugins/cases/server/client/cases/bulk_update.test.ts`?

I am thinking specifically about `'checks authorization for both reopenCase and updateCase operations when reopening a case'`.

This test only checks that `expect(clientArgs.authorization.ensureAuthorized).not.toThrow();` but if we had `expect(clientArgs.authorization.ensureAuthorized).toHaveBeenCalledWith(...)` we would have caught this bug.

Could you please assert it is called with the proper params?